### PR TITLE
[UXIT-1541] Event Page - Include calendar option

### PR DESCRIPTION
**Pull request by mirhamasala**
## 📝 Description

- **Type:** New feature

## 🛠️ Key Changes

- [Change 1 - Brief description]
- [Change 2 - Brief description]
- ...

## 📸 Screenshots

Attach if there are UI changes.

### CODE REVIEW

File: src/app/events/[slug]/page.tsx [hunk 1]
```diff
@@ -59,3 +65,20 @@ export default function EventEntry({ params }: EventProps) {
     </>
   )
 }
+
+function buildCtaArray(
+  externalLink: Event['externalLink'],
```
Comment (barbaraperic) [inline src/app/events/[slug]/page.tsx:70]: `externalLink` should be also optional right?

Maybe we could extract the type?

```
type EventCtaProps = {
  externalLink?: Event['externalLink']
  lumaCalendarLink?: Event['lumaCalendarLink']
}
```

File: src/app/events/[slug]/page.tsx [hunk 2]
```diff
@@ -59,3 +65,20 @@ export default function EventEntry({ params }: EventProps) {
     </>
   )
 }
+
+function buildCtaArray(
```
Comment (barbaraperic) [inline src/app/events/[slug]/page.tsx:69]: Should we move the function in a util file? In case we want to reuse it?

**Review by barbaraperic — APPROVED** [review]
Looks good, works well 👍  left a few tiny comments

File: src/app/events/[slug]/page.tsx (hunk 2 above)
Comment (mirhamasala) [reply src/app/events/[slug]/page.tsx:69]: In a local `utils` folder?

Comment (mirhamasala) [reply src/app/events/[slug]/page.tsx:69]: Yes! Good idea. Thank you.

File: src/app/events/[slug]/page.tsx (hunk 1 above)
Comment (mirhamasala) [reply src/app/events/[slug]/page.tsx:70]: Yes!

File: public/admin/config.yml [hunk 3]
```diff
@@ -668,5 +668,10 @@ collections:
         widget: "string"
         required: false
         hint: "External link to the event page"
+      - name: "luma-calendar-link"
+        label: "Luma Calendar URL"
+        widget: "string"
+        required: false
+        hint: "Luma Calendar URL for the event. Include relevant tags when appropriate, e.g., https://lu.ma/filecoin?tag=fil%20bangkok"
```
Comment (barbaraperic) [inline public/admin/config.yml:675]: Just curious - why adding the tag? We want to have the title of the event as a tag? Do they edit the url manually?

**Pull request merged**